server/athenian/api/internal/miners/github/repository.py

```diff
@@ -73,39 +73,34 @@ async def mine_repositories(
     assert isinstance(time_from, datetime)
     assert isinstance(time_to, datetime)
     repo_name_to_node = prefixer.repo_name_to_node.get
-    repo_ids = [repo_name_to_node(r) for r in repos]
+    # logical repos are not mapped to ids by prefixer and are ignored in mdb queries
+    repo_ids = [id_ for r in repos if (id_ := repo_name_to_node(r)) is not None]
```

Comment:
We can safely use the old code. `IN (..., null, ...)` works with ignoring the nulls. It's totally OK.

Reply:
reverted, left comment